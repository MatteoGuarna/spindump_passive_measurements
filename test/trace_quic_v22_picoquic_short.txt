Short test run on localhost with IPv6 and Picoquic, using version -22
