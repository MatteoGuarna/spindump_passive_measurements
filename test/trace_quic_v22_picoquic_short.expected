QUIC ::1 <-> ::1 aa73b2f7599246c508ceeb-null (49313:4443) at 1563724803062069 new starting packets 0 0 bytes 0 0
QUIC ::1 <-> ::1 aa73b2f7599246c508ceeb-ceeb1d4433d3537208047a (49313:4443) at 1563724803062443 change starting packets 1 0 bytes 1280 0 bandwidth 1280 0
QUIC ::1 <-> ::1 aa73b2f7599246c508ceeb-ceeb1d4433d3537208047a (49313:4443) at 1563724803062443 measurement up right 374 packets 1 0 bytes 1280 0 bandwidth 1280 0
